docs/examples/ex_extract.rst

```diff
@@ -14,11 +14,145 @@ modeled values.
 Regular Grid Models
 -------------------
 
-:py:func:`pysatModels.utils.extract.extract_modeled_observations` supports
-extracting values from models on a regular grid. The function can linearly
-interpolate model values onto instrument locations or use the nearest modeled
-location. Uses :py:func:`scipy.interpolate.interpn` as the underlying
-interpolation function.
+:py:func:`pysatModels.utils.extract.extract_modelled_observations` supports
+extracting values from models on a regular grid onto observed locations. The
+function can linearly interpolate model values onto instrument locations or use
+the nearest modeled location. Uses :py:func:`scipy.interpolate.interpn` as the
+underlying interpolation function. This function can handle either
+:py:mod:`pandas` or :py:mod:`xarray` formatted :py:class:`pysat.Instrument`
+observational data.  Because multi-dimensional data can be more complicated,
+let's use `Jicamarca ISR drift data <https://pysatmadrigal.readthedocs.io/en/latest/supported_instruments.html#jro-isr>`_ as an example.
+
+.. code:: python
+
+   import datetime as dt
+   import pandas as pds
+   import pysat
+   from pysatMadrigal.instruments import jro_isr
+   import pysatModels
+
+   stime = dt.datetime(2021, 1, 3)
+   jro = pysat.Instrument(inst_module=jro_isr, tag='drifts', user='Your Name',
+                          password='your.email@inst.type')
+
+    # Download data if necessary and then load it
+    if stime not in jro.files.files:
+        jro.download(start=stime)
+    jro.load(date=stime)
+
+    # Get fake model data from the pysat model test instrument
+    mod_drange = pds.date_range(stime, stime + dt.timedelta(days=1), freq='1D')
+    model = pysat.Instrument('pysat', 'testmodel', tag='',
+                             file_date_range=mod_drange)
+    model.load(date=stime)
+
+    # Check the loaded variables
+    print(jro.variables, model.variables)
+
+This yeilds:
+
+.. code:: python
+
+   ['time', 'gdalt', 'gdlatr', 'gdlonr', 'kindat', 'kinst', 'nwlos', 'range',
```

Comment:
I get a slightly different list since I am running `load_meta_proc`.

```
['time', 'gdalt', 'gdlatr', 'gdlonr', 'kindat', 'kinst', 'nwlos', 'range', 'vipn', 'dvipn', 'vipe', 'dvipe', 'vi7', 'dvi7', 'vi8', 'dvi8', 'paiwl', 'pacwl', 'pbiwl', 'pbcwl', 'pciel', 'pccel', 'pdiel', 'pdcel', 'jro10', 'jro11', 'year', 'month', 'day', 'hour', 'min', 'sec', 'spcst', 'pl', 'cbadn', 'inttms', 'azdir7', 'eldir7', 'azdir8', 'eldir8', 'jro14', 'jro15', 'jro16', 'ut1_unix', 'ut2_unix', 'recno'] ['uts', 'time', 'latitude', 'longitude', 'altitude', 'slt', 'mlt', 'dummy1', 'string_dummy', 'unicode_dummy', 'int8_dummy', 'int16_dummy', 'int32_dummy', 'int64_dummy', 'dummy2']
```

Reply:
Hmmm.... I don't see why that would make a difference.  I wonder if you are using the right data file.

Reply:
Also, this is really hard to read.  Can you specify where the differences are?

Reply:
make an issue over new variables in test Instrument.